server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityStatusAction.java

```diff
@@ -75,13 +74,15 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
```

Comment:
title of commit "SONAR-8716 fix check of permissions in api/ce/activity" is incorrect, it should be "SONAR-8716 fix check of permissions in api/ce/activity_status"